packages/inventory/config/webpack.constants.js

```diff
@@ -31,7 +32,7 @@ module.exports = {
             'react-dom': 'react-dom',
             redux: 'redux',
             axios: 'axios',
-            react: 'react'
+            react: 'customReact'
```

Comment:
Just to double-check. Does this mean that wen initializing inventory, the application will have to change the name of the react dependency when they call the `loadInventory` function?

Reply:
Nope, chrome does that for the app. I am working on a PR in chrome space.

Reply:
The chrome part can be found here https://github.com/RedHatInsights/insights-chrome/pull/856